.github/workflows/release.yml

```
@@ -29,12 +29,70 @@ jobs:
           :catalog:publishCatalogPublicationToSonatypeRepository -DLIBRARY_VERSION=${{ github.event.release.tag_name }}
           --max-workers 1 closeAndReleaseStagingRepository"
 
-      - name: Microsoft Teams Notification
-        uses: skitionek/notify-microsoft-teams@v1.0.8
-        if: success()
+      - name: Send Teams notification
+        uses: fjogeleit/http-request-action@v1
```

Could we use the latest version?
```suggestion
        uses: fjogeleit/http-request-action@v1.16.4
```